spec/models/equipment_model_spec.rb

```diff
@@ -151,6 +151,51 @@
     end
   end
 
+  describe 'counter cache' do
+    before(:each) do
+      @model = FactoryGirl.create(:equipment_model)
+      @res = FactoryGirl.build(:checked_out_reservation,
+                               equipment_model: @model, 
+                               start_date: Time.zone.today - 2.days,
+                               due_date: Time.zone.today - 1.day)
+      @res.save(validate: false)
+    end
+    it 'increments when a reservation is marked as overdue' do
+      @res.update_attribute(:overdue, true)
+      expect(@model.overdue_count).to eq 1
```

Comment:
Just a stylistic comment, I think we've been wrapping the parameters of expectation methods (e.g. `eq`)  in parentheses (so `expect(foo).to eq(0)`).